### DISCUSSION TITLE
Mistake. Self Pooling ---> Claim Rewards

### DISCUSSION THREAD

**Discussion by Scrooghy** [Chia-Network/chia-blockchain#11312] [Support]
Hello

Self Pooling ---> Claim Rewards ---> In the "Fee XCH" field, I mistakenly indicated not the commission for the transfer, but the amount of the transfer. How can this error be corrected? Help me pls.

**Comment by Scrooghy** [comment]
<a href="https://fastpic.org/view/117/2022/0427/58f28c1349468357cb81e0e04bd4c2f8.jpg.html" target="_blank"><img src="https://i117.fastpic.org/thumb/2022/0427/f8/58f28c1349468357cb81e0e04bd4c2f8.jpeg" border="0"></a>

Both addresses belong to me. But the balance has not increased. Please help me resolve this issue. Thank you very much in advance.

**Comment by loppefaaret** [comment]
I'm sorry - but completing a transaction with a fee, is not possible to reverse - best you can do is "delete unconfirmed transactions" in the wallet, and hope your transaction wasn't committed to the mempool of a node yet... but if it has gone through, there is no way to reverse it

**Reply by Scrooghy** [reply]
It's a pity! Such a stupid mistake. I'm probably the only one in the world who did this)) And how can I know committed my transaction to the mempool of a node or no?

**Reply by loppefaaret** [reply]
with a fee that large, it would instantly be included into the next block of any node that had the transaction in its mempool, and creating a block

**Comment by Scrooghy** [comment]
Let it be a small gift from me for the development of the network) and which commission is better to set when mining in the SELF Pooling?

**Reply by loppefaaret** [reply]
if you are in a rush, use what is equal to a couple of USD cents... if not, i suggest 100 mojos...

**Reply by Scrooghy** [reply]
thank you very much for your reply
